docs/api-types/service-import.md

```diff
@@ -0,0 +1,54 @@
+# ServiceImport API Reference
+
+## Introduction
+
+`ServiceImport` is a resource referring to a Service outside the cluster, paired with [`ServiceExport`](service-export.md)
```

Comment:
> paired with ServiceExport

Is it constrained to ServiceExport or any target group can be imported?

Reply:
this is a good point, I'll make a note of that use case.